helpers/test.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
             import pytest
             if not self.pytest_args:
                 self.pytest_args = ['sherpa',]
-            errno = pytest.main(self.pytest_args)
+            errno = pytest.main([self.pytest_args,])
```

Comment:
Nope, I shouldn't have pushed the commit before the PR builds ran. PR builds do not use the specific commit, but the hidden PR references, so when you update the PR while the builds are still running, the builds use the latest code. I'll remove the latest commit and rerun the builds.